charts/v3.0/cray-nls/Chart.yaml

```diff
@@ -0,0 +1,54 @@
+#
+# MIT License
+#
+# (C) Copyright [2022-2023] Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+apiVersion: v2
+name: "cray-nls"
+version: 3.0.0
+description: "Kubernetes resources for cray-nls"
+home: "https://github.com/Cray-HPE/cray-nls-charts"
+sources:
+  - "https://github.com/Cray-HPE/cray-nls"
+dependencies:
+  - name: cray-service
```

Comment:
Should this be?  Not certain but given it's for pg-only ...??
  - name: cray-postgresql
    version: ^1.0.0
    repository: "https://artifactory.algol60.net/artifactory/csm-helm-charts/"
    alias: cray-service-pg-only